pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency.slf4j.version>1.7.32</dependency.slf4j.version>
     <dependency.snakeyaml.version>1.29</dependency.snakeyaml.version>
     <dependency.spring.version>5.3.13</dependency.spring.version>
-    <dependency.zeebe.version>1.2.5</dependency.zeebe.version>
+    <dependency.zeebe.version>1.3.0-alpha3</dependency.zeebe.version>
```

Comment:
There is also `1.2.6` which is fixed as well. Just in case you don't want to necessarily update to an alpha version.

Reply:
Thanks! I know, I want to create a 1.3.0-alpha3 release for this project, hence the alpha version.